pkgs/development/libraries/science/math/tensorflow-lite/default.nix

```diff
@@ -1,203 +1,119 @@
+      # copy the libs and binaries into the output dir
+      cp ./bazel-bin/tensorflow/lite/c/libtensorflowlite_c.so $out/lib
+      cp ./bazel-bin/tensorflow/lite/libtensorflowlite.so $out/lib
+      cp ./bazel-bin/tensorflow/lite/tools/benchmark/benchmark_model $out/bin
+      cp ./bazel-bin/tensorflow/lite/tools/benchmark/benchmark_model_performance_options $out/bin
+
+      find . -type f -name '*.h' | while read f; do
+        path="$out/include/''${f/.\//}"
+        install -D "$f" "$path"
 
-  makefile = "tensorflow/lite/tools/make/Makefile";
-
-  preBuild =
-    let
-      includes =
-        lib.concatMapStringsSep
-          " "
-          (subdir: "-I $PWD/tensorflow/lite/tools/make/downloads/${subdir}")
-          [
-            "neon_2_sse"
-            "gemmlowp"
-            "absl"
-            "fp16/include"
-            "farmhash/src"
-            "ruy"
-            "cpuinfo"
-            "cpuinfo/src"
-            "cpuinfo/include"
-            "cpuinfo/deps/clog/include"
-            "eigen"
-          ];
-    in
-    ''
-      # enter the vendoring lair of doom
-
-      prefix="$PWD/tensorflow/lite/tools/make/downloads"
-
-      mkdir -p "$prefix"
-
-      tar xzf ${fft2d-src} -C "$prefix"
-
-      ln -s ${ruy-src} "$prefix/ruy"
-      ln -s ${gemmlowp-src} "$prefix/gemmlowp"
-      ln -s ${neon-2-sse-src} "$prefix/neon_2_sse"
-      ln -s ${farmhash-src} "$prefix/farmhash"
-      ln -s ${cpuinfo-src} "$prefix/cpuinfo"
-      ln -s ${fp16-src} "$prefix/fp16"
-      ln -s ${tflite-eigen} "$prefix/eigen"
-
-      # tensorflow lite is using the *source* of flatbuffers
-      ln -s ${flatbuffers.src} "$prefix/flatbuffers"
-
-      # tensorflow lite expects to compile abseil into `libtensorflow-lite.a`
-      ln -s ${abseil-cpp.src} "$prefix/absl"
-
-      # set CXXSTANDARD=c++17 here because abseil-cpp in nixpkgs is set as
-      # such and would be used in dependents like libedgetpu
-      buildFlagsArray+=(
-        INCLUDES="-I $PWD ${includes}"
-        CXXSTANDARD="-std=c++17"
-        TARGET_TOOLCHAIN_PREFIX=""
-        -j$NIX_BUILD_CORES
-        all)
+        # remove executable bit from headers
+        chmod -x "$path"
+      done
     '';
+  };
+
+  fetchAttrs = {
+    sha256 = bazelDepsSha256;
+  };
 
-  installPhase = ''
-    mkdir "$out"
+  PYTHON_BIN_PATH = pythonEnv.interpreter;
 
-    # copy the static lib and binaries into the output dir
-    cp -r ./tensorflow/lite/tools/make/gen/linux_${stdenv.hostPlatform.uname.processor}/{bin,lib} "$out"
+  dontAddBazelOpts = true;
+  removeRulesCC = false;
 
-    find ./tensorflow/lite -type f -name '*.h' | while read f; do
-      path="$out/include/''${f/.\//}"
-      install -D "$f" "$path"
+  postPatch = ''
+    rm .bazelversion
+  '';
+
+  preConfigure = ''
+    patchShebangs configure
+  '';
 
-      # remove executable bit from headers
-      chmod -x "$path"
-    done
+  configurePhase = ''
+    runHook preConfigure
+    ./configure
+    runHook postConfigure
   '';
```

Comment:
Isn't this the default configurePhase? Don't we just need to disable the python specific changes with a variable?

Reply:
I was able to get rid of the custom configure phase by making sure no args get passed to the configure script:

96e6dcafd14e0c26fa51b166914040e27269a7e0